doc/source/whatsnew/v0.23.2.txt

```diff
@@ -27,7 +27,9 @@ Performance Improvements
 - Improved performance of membership checks in :class:`CategoricalIndex`
   (i.e. ``x in ci``-style checks are much faster). :meth:`CategoricalIndex.contains`
   is likewise much faster (:issue:`21369`)
--
+- Improved performance of :meth:`HDFStore.groups` (and dependent functions like
```

Comment:
you don't need to say dependent functions, you can just list .keys

Reply:
There are a number of places that this hits (.keys, .info, .__iterate__, __len__, items, and those that call those functions.).  Should I ignore them, list the ones that I know of, or just drop the "dependent functions" part and keep it simple?

Reply:
on 2nd thought this is ok